src/services/i18nTesting.ts

```diff
@@ -0,0 +1,31 @@
+// initialize i18n cache
```

Comment:
getting a TS error when it is outside of the src folder.  I'm thinking we could add another convention like the ".test.ts" excludes to exclude ".fixture.ts" from the prod build.  thoughts?